packages/nautilus.rb

```diff
@@ -78,5 +78,6 @@ def self.install
   
   def self.postinstall
     system "glib-compile-schemas #{CREW_PREFIX}/share/glib-2.0/schemas/"
+    FileUtils.touch "#{CREW_DEST_HOME}/.gtk-bookmarks"
```

Comment:
Shouldn't this be `FileUtils.touch "#{HOME}/.gtk-bookmarks"`?  You would use `CREW_DEST_HOME` for packaged files in `def self.install`.

Reply:
Ah yes. I copied from slack.

Reply:
Fixed.